webservices/args.py

```diff
@@ -412,11 +412,18 @@ def make_seek_args(field=fields.Int, description=None):
         validate=validate.OneOf(['house', 'senate', 'president']),
         description='Office sought, either President, House or Senate.',
     ),
+    'period': fields.Bool(missing=False),
 }
 
 schedule_a_candidate_aggregate = {
     'candidate_id': fields.List(IStr, required=True, description=docs.CANDIDATE_ID),
     'cycle': fields.List(fields.Int, required=True, description=docs.RECORD_CYCLE),
+    'period': fields.Bool(missing=False),
```

Comment:
If the `period` flag is `True`, then records are aggregated over the relevant election period. For example, if the user requests data for a presidential candidate with `cycle=2012&period=true`, then the response will sum aggregates from 2010 and 2012. This flag has analogous behavior for all the endpoints touched by this patch.

This is probably a signal that the flag isn't named well--suggestions welcome.